plugin/metrics/vars/vars.go

```diff
@@ -14,21 +14,21 @@ var (
 		Subsystem: subsystem,
 		Name:      "requests_total",
 		Help:      "Counter of DNS requests made per zone, protocol and family.",
-	}, []string{"server", "zone", "proto", "family", "type"})
+	}, []string{"server", "zone", "proto", "family"})
```

Comment:
I think we still want to keep the `type` here. I can see wanting to remove it on the duration histogram, but it's useful here.